packages/playwright/jest.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   testMatch: ['**/*.test.*'],
   testPathIgnorePatterns: ['__playwrightTests__/*'],
+  coveragePathIgnorePatterns: ['<rootDir>/src/inBrowserPostProcessing.js'],
```

Comment:
Why do we want to opt this file out of code coverage? The CodeCov notification is meant to notify us of what is missing coverage not block PRs. I think it's a good thing to know if our tests are covering this scenario at all!

Reply:
This code gets `page.evaluate()`-d into Playwright (on the browser side) so I don't think we can detect its coverage very well. I started writing unit tests for the various functions in this file but realized I was mocking out a ton of things, so opted to have it indirectly covered by our end to end tests instead.

Reply:
If it's indirectly covered, then we should not have to ignore it here.

Reply:
I was unable to resolve the `cov_whatever` getting inserted into the file during jest tests, so I kept the coverage-ignore of this file. I was able to remove the lint-ignore of the file though!